pax-runtime/src/engine/expanded_node.rs

```diff
@@ -54,10 +54,10 @@ pub struct ExpandedNode {
     pub children: RefCell<Vec<Rc<ExpandedNode>>>,
 
     /// Each ExpandedNode has a unique "stamp" of computed properties
-    pub properties: Rc<RefCell<dyn Any>>,
+    pub properties: RefCell<Rc<RefCell<dyn Any>>>,
```

Comment:
Is this outer RefCell necessary given that we have a RefCell inside?

Reply:
it is because of how our factories currently return an Rc<RefCell<_>>. A larger refactor is needed for the engine stuff but didn't want to bite it off so did this to make it work.